mmtrack/evaluation/metrics/tao_metrics.py

```diff
@@ -170,23 +195,66 @@ def compute_metrics(self, results: List) -> Dict[str, float]:
                     val = float('{:.3f}'.format(float(v)))
                     eval_results[key] = val
 
+        if 'bbox' in self.metrics:
+            if lvis is None:
+                raise ImportError(
+                    'Please run'
+                    ' pip install git+https://github.com/lvis-dataset/lvis-api.git '  # noqa
+                    'to manually install lvis')
+
+            logger.info('Evaluating detection results...')
+            lvis_gt = LVIS(gt_results_path)
+            lvis_dt = LVISResults(lvis_gt, pred_det_results_path)
+            lvis_eval = LVISEval(lvis_gt, lvis_dt, 'bbox')
+            lvis_eval.params.imgIds = self.img_ids
+            lvis_eval.params.catIds = list(
+                self.dataset_meta['categories'].keys())
+            lvis_eval.evaluate()
+            lvis_eval.accumulate()
+            lvis_eval.summarize()
+            lvis_eval.print_results()
```

Comment:
Can we use the Lvis metric in https://github.com/open-mmlab/mmdetection/blob/dev-3.x/mmdet/evaluation/metrics/lvis_metric.py#L32 to eval the bbox ap?

Reply:
If so, we can only evaluate the track ap in tao_metric, make the tao_metric more neat

Reply:
I tried to use mmdet's metric, but it will lose two percentage points of mAP. I'm looking for the reason